### ISSUE TITLE
Strip trailing whitespace on save doesn't appear to trigger

### ISSUE THREAD

**Issue by walterdavis** [elia/avian-missing.tmbundle#21]
I found that I had to change the Semantic Class option from `callback.document.export` to `callback.document.will-save` (same as the Ensure trailing newline action) in order for this to have any effect. Would you like a PR to that effect?

**Comment by elia** [comment]
Sure! 

Can you please check first if the export mode is updating the file on the file-system?

You can verify that by saving and reopening the file to let TM pickup the changes. 

If that is true I would say that in the end is just a matter of preference, I personally switched back and forth on the two, but I ended up with both in `export` mode because having the space disappear under the cursor was bothering me.

And alas I don't think there's a way to parameterize it on a preference or env variable.

**Comment by walterdavis** [comment]
That's an excellent point. I was looking in the file in TM, and expecting the trailing spaces to disappear. I didn't try closing the file and re-opening, but I did try switching tabs, and that did not appear to work.

**Comment by walterdavis** [comment]
You're correct. Export does work if you close the file completely and re-open it again. Please let me know if you'd still like a PR.

**Comment by elia** [comment]
If you have time I'd be more than happy to update the readme and explain this behavior and how to change it

**Comment by elia** [comment]
If you have time I'd be more than happy to merge a PR updating the readme and explaining this behavior and how to change it
